docs/user-guide/how-to-deploy.rst

```diff
@@ -0,0 +1,51 @@
+:desc: How to deploy your Rasa Assistant with Docker or Kubernetes
+
+.. _deploying-your-rasa-assistant:
+
+Deploying your Rasa Assistant
+===============================
+
+.. edit-link::
+
+This page explains when and how to deploy an assistant built with Rasa. 
+It will allow you to make your assistant available to users and set you up with a production-ready environment.
+If you haven't used Rasa before, we'd recommend that you start with the :ref:`rasa-tutorial`.
+
+.. contents::
+   :local:
+
+
+When to deploy your assistant
+--------------------------------
+
+The best time to deploy your assistant and make it available to test users is once it can handle the most important happy paths, or is what we call a "minimum viable assistant".
```

Comment:
```suggestion
The best time to deploy your assistant and make it available to test users is once you have built a :ref:`minimum viable assistant <glossary>`, one that can handle the most important happy paths.
```

Reply:
I am not sure that is quite the right syntax, but link to the glossary

Reply:
I can link to the glossary page itself, but couldn't find a way to reference a specific item in the glossary  since it is not a section header.

Reply:
adding a reference in the glossary `.. _minimum-viable-assistant:` then referring to it with `minimum-viable-assistant` or `minimum viable assistant <minimum-viable-assistant>` didn't quite work

Reply:
Yeah, that is fine. I think I ran into the same issue

Reply:
linked